### ISSUE TITLE
[bug] How-to step description validation has conflicting information (1000 vs 700 chars)

### ISSUE THREAD

**Issue by asdFletcher** [ONEARMY/community-platform#2627]
**Describe the bug**

The description field for creating a how-to step has a message and character counter that indicates to the user that they must have fewer than **1000** characters,  when in reality the javascript form validation is checking for maximum **700**. 

_This is my first issue, hoping to adopt this issue as my first PR :)_

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to `http://localhost:3000/how-to/create`
2. Scroll down to Step 1
3. Focus the description field and type 701 characters
4. The form validation indicators will conflict, and the form validation will fail

**Expected behaviour**
- The validation text should display to the user the actual number used for validation (either 1000 or 700), whichever is decided as the correct value
- The constant `HOWTO_STEP_DESCRIPTION_MAX_LENGTH` should be used in validation instead of a hard-coded number

**Screenshots**
Here, the form is **invalid** with 701 characters:
![image](https://github.com/ONEARMY/community-platform/assets/42685363/405b09c7-d665-4682-a0aa-95f83b8e9cec)

**Additional context**
Here there is the source of the inconsistency: `src/pages/Howto/Content/Common/HowtoStep.form.tsx`
![image](https://github.com/ONEARMY/community-platform/assets/42685363/1f8d6b44-0857-4a0c-acb1-93cde1e80e11)

**Comment by asdFletcher** [comment]
I believe the only blocker completing the work on this is deciding whether the limit should be 1000 characters or 700 characters

**Comment by iSCJT** [comment]
Thanks for picking up on this @asdFletcher.  It was a recent change to move from 1000 characters down to 700 but it was a mix up in the requirements we received and it should have stayed at 1000 and just been the addition of the minimum character limit that was added.

The second screw up was entirely my fault 🤦‍♂️.  I approved the pr that made the change and as you've seen the limits are now hardcoded rather than coming from the variable.

If you're happy to do a pr to sort this out for us that would be amazing 🤩
